app/code/core/Mage/Api/Model/Server/Adapter/Jsonrpc.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Magento
+ *
+ * NOTICE OF LICENSE
```

Comment:
btw you removed that line in one PR for all files. maybe don't add it here 🤔

Reply:
hahahaa right, this PR was much older than the other one :-) I'll change this one if the other gets approved :-) but thanks for the comment so that I won't forget!

btw I'm positive this PR will stay unapproved forever...